pkgs/os-specific/linux/displaylink/default.nix

```diff
@@ -44,6 +44,7 @@ in stdenv.mkDerivation rec {
       main systemd "$out/lib/udev/rules.d/99-displaylink.rules" "$out/lib/udev/displaylink.sh"
     )
     sed -i '2iPATH=${systemd}/bin:$PATH' $out/lib/udev/displaylink.sh
+    sed -i 's,systemctl start,systemctl start --no-block,g' $out/lib/udev/displaylink.sh
```

Comment:
Can you replace both `sed` commands by a proper patch that, uses substituteAll for the dynamic parts? https://github.com/NixOS/nixpkgs/blob/c19083961d4bb271d9aa63d61f855683137bba4b/pkgs/development/python-modules/pyocr/default.nix#L21